orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorQueuedJenkinsJobTask.groovy

```diff
@@ -59,4 +64,48 @@ class MonitorQueuedJenkinsJobTask implements OverridableTimeoutRetryableTask {
       throw e
     }
   }
+
+  @Override
+  long getBackoffPeriod() {
+    return 10000
+  }
+
+  @Override
+  long getTimeout() {
+    return TimeUnit.HOURS.toMillis(2)
+  }
+
+  /**
+   * Update the description of a running Jenkins build and provide a deep link back to the particular execution in
+   * Spinnaker.
+   *
+   * Requires that `wwwHost` be specified _and_ that the build is running (vs. queued).
+   */
+  private void updateBuildDescription(StageExecution stageExecution,
+                                      String jenkinsController,
+                                      String jobName,
+                                      Integer buildNumber) {
+    if (wwwHost == null) {
```

Comment:
Do we want an `info` log here potentially to inform existing operators that there's a handy feature that could be unlocked with some additional config?

Reply:
I was going to add something to https://spinnaker.io/setup/ci/jenkins/, a `log.info()` might not hurt as well.